hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/RandomOps.java

```diff
@@ -211,6 +147,8 @@ private HapiSpec retryLimitDemo() {
                         cryptoTransfer(tinyBarsFromTo(GENESIS, FUNDING, 7L)));
     }
 
+    // Since the freeze tests all shut down the network, they can’t be part of the regular hapi test suite. It was
+    // decided to exclude them from the hapi suite.
```

Comment:
Possibly add the following here in the next PR...:
`// @todo('8687') Revisit once isolated tests are enabled`

Reply:
done thanks